apps/cadence_voltus/build_voltus.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# TODO: install directory
+INSTALL_DIR="/netapps/voltus/"
```

Comment:
Is this correct?  It's not a default path that is created

Reply:
I made change to /voltus to make less confusion.

Reply:
if you make this /apps/voltus it will be exported to all the nodes for the standard nfsserver setup